scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/instances/CoderInstances.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.bigquery.instances
+
+import com.spotify.scio.coders.Coder
+import com.google.api.services.bigquery.model.TableRow
+import org.apache.beam.sdk.io.gcp.bigquery.TableRowJsonCoder
+
+trait CoderInstances {
+  implicit def tableRowCoder: Coder[TableRow] = Coder.beam(TableRowJsonCoder.of())
```

Comment:
Maybe be more source-compat if we make this available via `import com.spotify.scio.bigquery._`?

Reply:
Hi @nevillelyh right, tbh I thought about it; I avoided that because it deviates a bit from the overall way to add coders; in most modules now (including this one) `Coder`'s are brought into scope through `import com.spotify.scio.<smth>.coders._`. 

Since we also set explicitly the coder for the `TableRow` based IO's should cover most of the cases. wdyt? should we still add it?

On a side note, I'm thinking of removing the `GenericJson` coder from scope. Json encode/decode tends to be different and specific to IO's.

Reply:
Yeah should probably be enough.
Maybe we can document all the available coders/artifacts/namespaces in a table?

Reply:
That's a good idea!